server/api/db/base.py

```diff
@@ -61,6 +61,16 @@ def store_run(
     ):
         pass
 
+    def create_or_get_run(
```

Comment:
why not `get_or_create_run` like `get_or_create_project`

Reply:
thanks. I wanted to emphasis that we 1st do a create and then we get the run so I reversed the name